### PR TITLE
Login form new desing

### PR DESCRIPTION
**Pull request by YisusJe**
### Story: https://app.asana.com/0/1191220738645130/1197128960505729 | https://app.asana.com/0/1191220738645130/1198180846770457

new login

### CODE REVIEW

File: src/components/LoginForm/index.js [hunk 1]
```diff
@@ -186,22 +169,47 @@ const LoginFormUI = (props) => {
                   })}
                   onChange={(e) => hanldeChangeInput(e)}
                 />
+                <ForgotPassword>
+                  {t('FORGOT_YOUR_PASSWORD', 'Forgot your password?')} {elementLinkToForgotPassword}
+                </ForgotPassword>
                 <Button color='primary' type='submit' disabled={formState.loading}>
```
Comment (quirozcarlos) [inline src/components/LoginForm/index.js:175]: Same here

File: src/components/LoginForm/index.js [hunk 2]
```diff
@@ -186,22 +169,47 @@ const LoginFormUI = (props) => {
...
+              {t('NEW_ON_PLATFORM', 'New on Ordering?')}
+              <a href={linkToSignup}>{t('CREATE_AN_ACCOUNT', 'Create an account')}</a>
+            </>
+          )}
+        </NewOnPlatform>
+        {/* <SocialIcons>
+          {configs?.facebook_id && <FacebookLoginButton ordering={ordering} appId={configs.facebook_id.value} handleSuccessFacebookLogin={handleSuccessFacebook} />}
+          {/* <FaApple />
+                  <AiOutlineGoogle />
+        </SocialIcons> */}
```
Comment (quirozcarlos) [inline src/components/LoginForm/index.js:204]: Why this comment?

Comment (YisusJe) [reply src/components/LoginForm/index.js:204]: This is for the mobile design

Comment (quirozcarlos) [reply src/components/LoginForm/index.js:204]: ok, if it's unused code, please delete, in another case, add comment description with `/** --comment-- */` notation.

**Pull request merged**